packages/mint-components/src/components/sqm-referral-card/sqm-referral-card.feature

```diff
@@ -35,3 +35,44 @@ Feature: Referral Card
 			| end    | the bottom |
 			| N/A    | the top    |
 
+	@ui
+	Scenario Outline: Container border can be toggled
+		Given a user is viewing the Referral Card component
+		And the prop "remove-border" has <value>
+		Then the referral card component <maybe> includes a border
+		Examples:
+			| value | maybe    |
+			| true  | does     |
+			| false | does not |
+
```

Comment:
@noahwc did we add the ability for it to only have one slot in the end?

Reply:
Yes we did, there is now a spec outlining behaviour if there is no content in one column